awebox/opti/initialization_dir/induction.py

```diff
@@ -60,31 +62,31 @@ def initial_guess_general(init_options, nlp, formulation, model, V_init):
 
 
 def initial_guess_vortex(init_options, nlp, formulation, model, V_init):
+
     if not nlp.discretization == 'direct_collocation':
-        awelogger.logger.error('vortex induction model is only defined for direct-collocation model, at this point')
+        message = 'vortex induction model is only defined for direct-collocation model, at this point'
+        awelogger.logger.error(message)
```

Comment:
at this point you might not only want to log an error but raise an exception

Reply:
agreed! thank you!